lib/bitters/generator.rb

```diff
@@ -16,11 +16,12 @@ def install
         puts "Bitters files already installed, doing nothing."
       else
         install_files
-        puts "Bitters files installed to #{install_path}/base"
+        puts "Bitters files installed to #{install_path}/"
```

Comment:
I would remove the trailing slash, if you specify the path to be `custom_path/` it will be printed as `...installed to custom_path//`